tracer/src/Datadog.Trace/Tracer.cs

```diff
@@ -647,6 +647,11 @@ private static string GetApplicationName()
             {
                 try
                 {
+                    if (AzureAppServices.Metadata.IsRelevant)
+                    {
+                        return AzureAppServices.Metadata.SiteName;
+                    }
```

Comment:
This should have been moved up above this inner `try` because the log message does not apply.